bcda/main.go

```diff
@@ -439,19 +458,73 @@ func removeExpired(hrThreshold int) {
 
 			id := int(j.ID)
 			jobDir := fmt.Sprintf("%s/%d", os.Getenv("FHIR_PAYLOAD_DIR"), id)
-			expDir = fmt.Sprintf("%s/%d", os.Getenv("FHIR_EXPIRED_DIR"), id)
+			expDir = fmt.Sprintf("%s/%d", os.Getenv("FHIR_ARCHIVE_DIR"), id)
```

Comment:
not a big deal but this name should probably change too.

Reply:
I left it there because I wasn't sure if it had been used anywhere. I plan to delete it after I'm sure no clean up needs to happen (kind of like a database migration, but in reverse).

Reply:
makes sense